textalyze.js

```diff
@@ -33,17 +36,15 @@ const getHistogram = (frequencies, totalChartSize) => {
     const chartBar = Array(totalChartSize)
       .fill(bar, 0, itemBars)
       .join('');
+    const itemBarSpace = (frequency >= 0.1) ? '' : ' ';
 
-    result.push(`${letter} [ ${(frequency * 100).toFixed(2)}% ] ${chartBar}>`);
+    result.push(`${letter} [${itemBarSpace}${(frequency * 100).toFixed(2)}%] ${chartBar}>`);
```

Comment:
This pattern is good for `map`, but I understand why you didn't use it here: the `Map` type doesn't have a `map` method.  I find that incredibly annoying.

You could define a method that produces a single line, say, `getHistogramLine`, and then:
```javascript
function getHistogram(frequencies) {
  return Array.from(frequencies).map(getHistogramLine).join('\n');
}
```

Reply:
Great, that really annoyed me too. 

Didn't know about that Array.from approach to convert the Map object